content/_data/changelogs/weekly.yml

```diff
@@ -8210,6 +8210,249 @@
       authors:
         - wadeck
 
+- version: "2.246"
+  date: 2020-07-21
+  changes:
+  - type: major rfe
+    category: major rfe
+    pull: 4823
+    references:
+      - pull: 4823
+      - issue: 60005
+      - issue: 61862
+      - url: https://www.jenkins.io/doc/administration/requirements/windows/
+        title: Windows support policy
+    authors:
+    - NextTurn
+    - oleg-nenashev
+    message: |-
+      Stop supporting .NET Framework 2.0 for launching Jenkins server and agents as a Windows service.
+      .NET Framework 4.0 or above is now required.
+  - type: major rfe
+    category: major rfe
+    pull: 4823
+    references:
+      - pull: 4823
+      - url: https://github.com/winsw/winsw/releases
+        title: Windows Service Wrapper changelog
+      - url: https://github.com/jenkinsci/windows-slave-installer-module/releases/tag/windows-slave-installer-2.0
+        title: Windows Agent Installer 2.0 changelog
+    authors:
+    - NextTurn
+    - oleg-nenashev
+    message: |-
+      Update Windows Service Wrapper from 2.3.0 to 2.9.0.
+      Includes numerous improvements and bugfixes.
+      Most notably, the service installer will now ask for permission elevation if the required permissions are missing.
+  - type: rfe
+    category: rfe
+    pull: 4852
+    authors:
+    - oleg-nenashev
+    message: |-
+      Reorder configuration and delete buttons.
+  - type: rfe
+    category: rfe
+    pull: 4683
+    issue: 62014
+    authors:
+    - daniel-beck
+    message: |-
+      Add the ability to filter out environment variables for Shell and Windows batch build steps.
+  - type: rfe
+    category: rfe
+    pull: 4835
+    issue: 63002
+    authors:
+    - fqueiruga
+    message: |-
+      Revamp the tabs and tables with better spacing and a consistent color palette.
+  - type: rfe
+    category: rfe
+    pull: 4748
+    issue: 61806
+    authors:
+    - 95jonpet
+    - jglick
+    message: |-
+      Remove footer timestamp.
+  - type: rfe
+    category: rfe
+    pull: 4840
+    issue: 25259
+    authors:
+    - 'Tidhar Klein Orbach (tidhar.o@taboola.com)'
+    message: |-
+      Add sorting to old data management tables.
+  - type: rfe
+    category: rfe
+    pull: 4842
+    issue: 62747
+    authors:
+    - timja
+    message: |-
+      Add alert-success banner.
+  - type: rfe
+    category: rfe
+    pull: 4834
+    issue: 63022
+    authors:
+    - stellargo
+    message: |-
+      Allow fingerprint storage engine to be selected from the configuration page.
+  - type: rfe
+    category: rfe
+    pull: 4073
+    issue: 59136
+    authors:
+    - jetersen
+    - daniel-beck
+    message: |-
+      Plugins marked as <b>deprecated</b> in update site metadata will now have a banner about it, and if any are installed, a notification is shown to admins.
+  - type: rfe
+    category: rfe
+    pull: 4820
+    issue: 56109
+    authors:
+    - timja
+    message: |-
+      Minor styling tweaks in preparation for changing form layout.
```

Comment:
Would be nice to expand so that it is more helpful to users or to move to "Internal". Right now it is not possible to sat what us the change about without looking into the code